includes/reader-activation/class-reader-activation.php

```diff
@@ -679,7 +686,7 @@ public static function is_user_reader( $user, $strict = false ) {
 			 *
```

Comment:
Move the filter doc along with the filter

Reply:
Done in 2eb469db70358ffc9d417e39f299768d4a0d3e0e